resources/lang/en/applicant/application_template.php

```diff
@@ -101,9 +101,15 @@
     'veterans_title' => 'Veterans Claim',
     'veterans_content' => 'Are you a veteran or a member of the Canadian Armed Forces?',
     'veterans_label' => 'Select One',
-    'language_title' => 'Language Requirement',
-    'language_copy' => 'Which language would you prefer for this application process?',
-    'language_label' => 'Select One',
+    'director_name_label' => 'Director Name',
+    'director_title_label' => 'Director Title',
+    'director_email_label' => 'Director Email',
+    'reference_name_label' => 'Reference Name',
+    'reference_title_label' => 'Reference Title',
+    'reference_email_label' => 'Reference Email',
+    'required_info_heading' => 'Required Information',
+    'director_copy' => 'Director',
```

Comment:
Lets adapt the full question text: `What is your current director’s name and email? (We’ll need approval that you can be loaned to another team for a bit to help out.)`
And for the reference copy: `What is the name and email of an additional reference who can confirm your skill set? (Preferably a current or recent Government of Canada supervisor.)`

Reply:
Adapted.